File: core/src/org/rexellentgames/dungeon/debug/GenerateCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.Dungeon;

public class GenerateCommand extends ConsoleCommand {
	{
		name = "/reset";
		shortName = "/rst";
		description = "Generates a new game and player (might freeze)";
	}

	@Override
	public void run(Console console, String[] args) {
		Dungeon.newGame();
	}
}

File: core/src/org/rexellentgames/dungeon/debug/GenerateCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.Dungeon

class GenerateCommand : ConsoleCommand() {
    init {
        name = "/reset"
        shortName = "/rst"
        description = "Generates a new game and player (might freeze)"
    }

    override fun run(console: Console, args: Array<String>) {
        Dungeon.newGame()
    }
}